com.unity.postprocessing/PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -723,6 +724,7 @@ void OnPostRender()
                                 m_Camera.projectionMatrix = m_Camera.GetStereoProjectionMatrix(Camera.StereoscopicEye.Left);
```

Comment:
This issue was introduced by the temporalAA projection matrix fix from @fcoulombe. Fyi, all XRSettings calls should be #if with (ENABLE_VR_MODULE && ENABLE_VR) (like in line 8 of PostProcessLayer).